src/styles/card.scss

```diff
@@ -0,0 +1,91 @@
+.card {
+  margin: 0 auto;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  background-color: $white--color;
+  border: 1px solid transparent;
+  border-radius: 5px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+
+    & .img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+    }
+  }
+
+  &-top {
+    margin-bottom: 40px;
+  }
+
+  &__title {
+    line-height: 18px;
+    font-weight: 500;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-size: 10px;
+    color: $secondary--color;
+    margin-bottom: 4px;
+  }
+
+  &__rate {
+    display: flex;
+    align-items: center;
+    justify-content: space-between;
+    margin-bottom: 12px;
+  }
+
+  &__reviews {
+    padding-top: 1px;
+    font-size: 10px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+
+    & .text {
+      line-height: 18px;
+      color: $secondary--color;
+    }
+
+    & .value {
+      line-height: 18px;
+      font-size: 16px;
+      font-weight: 700;
+    }
+  }
+
+  &__link {
+    margin: 0 auto;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
+    height: 38px;
+    background-color: $blue--accent;
+    color: $white--color;
+    line-height: 16px;
+    font-size: 14px;
+    font-weight: 700;
+    border-radius: 5px;
+    text-transform: uppercase;
+    border: 1px solid transparent;
+    transition: all 0.5s ease;
```

Comment:
Don't set all for transition. Better specify the exact properties
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x